### PR TITLE
Use OPENEDX_RELEASE instead of release branch if specified

### PR DESCRIPTION
**Pull request by singingwolfboy**
@clytwynec Is this what you were looking for?

### CODE REVIEW

**Comment by clytwynec** [comment]
Yea.  Do you know if we will be continuing to use the 'release' branch for other reasons after named releases start?  If not, we should make the default 'master'.

**Comment by singingwolfboy** [comment]
We will continue to use the `release` branch, even after we've pushed out our first named release. The `release` branch tracks the weekly releases to edx.org, while the named releases are going to be much less frequent than that.

**Comment by fredsmith** [comment]
:+1:

File: vagrant/release/devstack/Vagrantfile [hunk 1]
```diff
@@ -21,14 +22,14 @@ cd /edx/app/edx_ansible/edx_ansible/playbooks
 # The vagrant-devstack.yml playbook will also do this, but only
 # after loading the playbooks into memory.  If these are out of date,
 # this can cause problems (e.g. looking for templates that no longer exist).
-/edx/bin/update configuration release
+/edx/bin/update configuration ${OPENEDX_RELEASE-release}
```
Comment (feanil) [inline vagrant/release/devstack/Vagrantfile:25]: Don't you need a `:` before the dash?

Comment (singingwolfboy) [inline vagrant/release/devstack/Vagrantfile:25]: [Nope.](http://www.tldp.org/LDP/abs/html/parameter-substitution.html) "${parameter-default} and ${parameter:-default} are almost equivalent. The extra : makes a difference only when parameter has been declared, but is null."

Comment (jarv) [inline vagrant/release/devstack/Vagrantfile:25]: I prefer extra if statements to inscrutable shell syntax personally, but this works.

**Pull request merged**